README.md

```diff
@@ -16,8 +16,33 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Avoid using `.css()`](#avoid-using-css)
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Avoid using `.css()`
+jQuery can get and set styling directly on an element with the [`.css()`](https://api.jquery.com/css/) method.
+When using `.css()` to set CSS it will set the styles inline and you are mixing concerns (styling and logic).
+
+### Why?
+* By toggling classes the same thing can be accomplished.
+* Separation of concerns, don't mix CSS with JavaScript.
+* When the `.css()` method is not used it can be omitted when creating a custom jQuery build.
+
+### How?
+``` javascript
+/* avoid: mixing JavaScript and CSS */
+$('[my-element]').css('border', '1px solid green');
+
+/* recommended: using a class */
```

Comment:
maybe split these two JS snippets so that the second one belongs more with the CSS?